assets/site.css

```diff
@@ -40,6 +40,20 @@ a:hover {
     padding-top: 2rem;
 }
 
+/*
+ * Sticky sidebar menu!
+ */
+.sidebar-sticky {
+    height: 100%;
+    width: 200px;
+    position: fixed;
+    z-index: 900;
+    top: 1rem;
+    overflow-x: hidden;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+}
```

Comment:
Heh, seems we're already using this non-existent class in layout.html... I guess this is left-over from some previous version...

Reply:
The class was there before, and its a good idea, so I tried to fulfil the functionality.

Reply:
Yeah, I get that. I was just puzzled that we're using an undefined class. Do you know if any linters pick up things like this?

Reply:
Funny that you ask, I've built a tool 😄  that can remove unused CSS by comparing the input CSS with the CSS used in HTML: https://www.npmjs.com/package/css-purge